hack/update-deps.sh

```diff
@@ -95,63 +53,5 @@ function contour_operator_yaml() {
 
 rm -rf config/contour/*
 
-# We do this manually because it's challenging to rewrite
-# the ClusterRoleBinding without collateral damage.
-cat > config/contour/internal.yaml <<EOF
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
-metadata:
-  name: knative-contour-internal
-  labels:
-    networking.knative.dev/ingress-provider: contour
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
-  name: $CLUSTER_ROLE_NAME
-subjects:
-- kind: ServiceAccount
-  name: contour
-  namespace: contour-internal
----
-EOF
-
-contour_yaml \
-  | rename_cluster_role $CLUSTER_ROLE_NAME \
-  | delete_contour_cluster_role_bindings \
-  | rewrite_contour_namespace contour-internal \
-  | rewrite_serve_args contour-internal | rewrite_user \
-  | rewrite_image | rewrite_image_pull_policy | rewrite_command | disable_hostport | privatize_loadbalancer \
-  | add_ingress_provider_labels  >> config/contour/internal.yaml
-
-# We do this manually because it's challenging to rewrite
-# the ClusterRoleBinding without collateral damage.
-cat > config/contour/external.yaml <<EOF
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
-metadata:
-  name: knative-contour-external
-  labels:
-    networking.knative.dev/ingress-provider: contour
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
-  name: $CLUSTER_ROLE_NAME
-subjects:
-- kind: ServiceAccount
-  name: contour
-  namespace: contour-external
----
-EOF
-
-contour_yaml \
-  | rename_cluster_role $CLUSTER_ROLE_NAME \
-  | delete_contour_cluster_role_bindings \
-  | rewrite_contour_namespace contour-external \
-  | rewrite_serve_args contour-external | rewrite_user \
-  | rewrite_image | rewrite_image_pull_policy | rewrite_command | disable_hostport \
-  | add_ingress_provider_labels >> config/contour/external.yaml
-
-rm -rf config/contour-operator/operator.yaml
-
-contour_operator_yaml \
-  >> config/contour-operator/operator.yaml
+contour_yaml | run_ytt --ignore-unknown-comments --data-value namespace=contour-internal --data-value clusterrole.name=$CLUSTER_ROLE_NAME -f hack/overlays -f - >> config/contour/internal.yaml
```

Comment:
can we format the args on a new line so it's easier to read the whole command

```suggestion
contour_yaml | \
  run_ytt --ignore-unknown-comments \
    --data-value namespace=contour-internal \
    --data-value clusterrole.name=$CLUSTER_ROLE_NAME \
    -f hack/overlays \
    -f - >> config/contour/internal.yaml
```